test/helper.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ZONE="us-west1-a"
```

Comment:
It's usually a bad idea to define globals in a util.sh that is mostly functions.
It's better to pass these as arguments to the function.

**IF** you are using global variables, you need to document them in the bash function comments, which I don't see consistently below.